sdk/forms/vite.config.ts

```diff
@@ -1,111 +1,80 @@
-import {defineConfig} from 'vite';
+import { defineConfig, UserConfig, UserConfigFnPromise } from 'vite';
 import * as path from 'path';
-import {Vault} from "@piiano/testcontainers-vault";
-import pkg from './package.json';
+import { optimizeVaultClientBundleSize } from './vite/optimize-vault-client-bundle-size';
+import { singleInlinedHtml } from './vite/single-inlined-html';
+import { appendSuffixes } from './vite/append-suffixes';
+import { initDevelopmentVault } from './vite/init-dev-vault';
+import { version } from './package.json';
 
 // https://vitejs.dev/config/
-export default defineConfig((async () => {
-  let devVaultPort = 8123;
-  if (process.env.NODE_ENV === 'development') {
-    devVaultPort = await initDevelopmentVault();
+const config: UserConfigFnPromise = async ({ mode }): Promise<UserConfig> => {
+  if (mode === 'dev') {
```

Comment:
@ori-shalom Would this be publicly available?

Reply:
WDYM? This is only vite config used for bundling and local dev server. It's not delivered with the final artifacts.